src/dialogs/more-info/controls/more-info-update.ts

```diff
@@ -0,0 +1,193 @@
+import "@material/mwc-button/mwc-button";
+import "@material/mwc-linear-progress/mwc-linear-progress";
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import { supportsFeature } from "../../../common/entity/supports-feature";
+import "../../../components/ha-checkbox";
+import "../../../components/ha-formfield";
+import "../../../components/ha-markdown";
+import { UNAVAILABLE_STATES } from "../../../data/entity";
+import {
+  updateIsInstalling,
+  UpdateEntity,
+  UPDATE_SUPPORT_BACKUP,
+  UPDATE_SUPPORT_INSTALL,
+  UPDATE_SUPPORT_PROGRESS,
+  UPDATE_SUPPORT_SPECIFIC_VERSION,
+} from "../../../data/update";
+import type { HomeAssistant } from "../../../types";
+
+@customElement("more-info-update")
+class MoreInfoUpdate extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: UpdateEntity;
+
+  protected render(): TemplateResult {
+    if (
+      !this.hass ||
+      !this.stateObj ||
+      UNAVAILABLE_STATES.includes(this.stateObj.state)
+    ) {
+      return html``;
+    }
+
+    const skippedVersion =
+      this.stateObj.attributes.skipped_version ===
+      this.stateObj.attributes.latest_version;
+
+    return html`
+      ${this.stateObj.attributes.in_progress
+        ? supportsFeature(this.stateObj, UPDATE_SUPPORT_PROGRESS) &&
+          typeof this.stateObj.attributes.in_progress === "number"
+          ? html`<mwc-linear-progress
+              .progress=${this.stateObj.attributes.in_progress / 100}
+              buffer=""
+            ></mwc-linear-progress>`
+          : html`<mwc-linear-progress indeterminate></mwc-linear-progress>`
+        : ""}
+      <h3>${this.stateObj.attributes.title}</h3>
```

Comment:
If there is no title, will this render still the margin of h3 ?

Reply:
It will render the h3 element, but it will not take any place
Second example shows that <https://623851bf4fdc510008f01998--home-assistant-gallery.netlify.app/#more-info/update>
But I can add this conditionally if you want